docs/integrations/sources/amazon-seller-partner-migrations.md

```diff
@@ -1,5 +1,10 @@
 # Amazon Seller Partner Migration Guide
 
+## Upgrading to 3.0.0
```

Comment:
please update to follow these guidelines: https://docs.google.com/document/d/1V3ybA24Lj_fy7IQwsVVzSxs46qGOsUydwlAQEYSQfcE/edit

Reply:
Updated